### ISSUE TITLE
cannot go install

### ISSUE THREAD

**Issue by J-Zeitler** [modfin/zdap#8]
When I issue `go install github.com/modfin/zdap/cmd/zdap@latest` I get the following error:

```
go: downloading github.com/modfin/zdap v0.0.0-20230914134107-9ad7496f289e
go: github.com/modfin/zdap/cmd/zdap@latest (in github.com/modfin/zdap@v0.0.0-20230914134107-9ad7496f289e):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

offending line?

https://github.com/modfin/zdap/blob/master/go.mod#L53

**Comment by J-Zeitler** [comment]
package is unfortunately still called `github.com/bicomsystems/go-libzfs` so can't get rid of the replace

https://github.com/ubuntu/go-libzfs/blob/master/go.mod#L1C8-L1C41

**Comment by larsve** [comment]
I think the only fast workaround, is to clone the repo and build zdap locally, until we have removed the `replace` directive from `go.mod`.

**Comment by larsve** [comment]
This have been fixed as part of #12, closing this now.
